README.md

```diff
@@ -5,4 +5,117 @@
 [![Stackage Lts](http://stackage.org/package/elm-street/badge/lts)](http://stackage.org/lts/package/elm-street)
 [![Stackage Nightly](http://stackage.org/package/elm-street/badge/nightly)](http://stackage.org/nightly/package/elm-street)
 
-Crossing the road between Haskell and Elm
\ No newline at end of file
+Crossing the road between Haskell and Elm.
+
+## Examples
+
+Below you can see some examples of how Haskell data type are converted to Elm
+types with the `elm-street` library.
+
+### Records
+
+**Haskell**
+
+```haskell
+data User = User
+    { userName :: Text
+    , userAge  :: Int
+    }
+```
+
+**Elm**
+
+```elm
+type alias User =
+    { name : String
```

Comment:
What if user had `userId` field? How it would be in `Elm` if the `id` name id not allowed?

Reply:
In Elm `id` is actually called `identity` so we can have `id` field name

* https://package.elm-lang.org/packages/elm/core/latest/Basics#identity

Reply:
Oh, okay, sorry!